### DISCUSSION TITLE
No puedo "pushear"

### DISCUSSION THREAD

**Discussion by RoJerez** [INGCOM-UNRN-P1/cursada-2023#12] [Preguntas y respuestas] [answered]
Hola profe, cuando quiero hacer el paso del push me salta este error 
![push](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/134402809/821dc848-13a4-43c4-83dd-7a6f962084c0)

**Comment by martinvilu** [comment, marked as answer]
Naturalmente, no podes, ya que tenés que usar el repositorio que se crea con tu nombre y usuario.

El de la cursada, es para los materiales e información de la cursada exclusivamente.

**Reply by Lautymendez** [reply]
Buenas, me pasa lo mismo, estoy intentando terminar el tp 0 para subir el compilador.txt pero no me deja(recien estoy aprendiendo a usar el git bash je) . La diferencia es que estoy intentando subir a mi repositorio del TP0, me pide una autentificación y ya la complete pero de todas formas me tira error.
![error](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141681157/e7ef2e38-4abf-4650-bbe2-ababcd3a61c9)

**Reply by martinvilu** [reply]
Lo que te esta diciendo es que hay cambios en internet que no estan en tu computadora.

Para evitar esto que hace de `git` mas complicado, hay que hacer los cambios en un solo lugar, y si no es la computadora, hacer un `git pull` antes.

**Reply by Lautymendez** [reply]
Puse previamente el comando git pull y ahora me sale otro error
![error pull](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141681157/9bb30825-1add-4e83-9eb3-c69a9e6faf38)

**Reply by Lautymendez** [reply]
Pude solucionar el error! Consulta, una vez que se subio el archivo "compilador.txt" y la ficha, ¿ya esta completo el tp 0 ? ¿Lo recibio?

**Reply by martinvilu** [reply]
¿Lo cargaste en el formulario de entrega? Se complica así suelto vistes

**Reply by Lautymendez** [reply]
Nono, ahora me fijo en la bibliografía para ver como se entrega. Creo que había visto algo de eso sino me equivoco...

**Reply by martinvilu** [reply]
completa esto

https://docs.google.com/forms/d/e/1FAIpQLSfg_OUJ-rBTNLxhTuQkqg5jN4bl-ELa6g3u7LOjAj4HTBD6mg/viewform

**Reply by martinvilu** [reply]
Solo una cosa @Lautymendez 

Para evitar que tu pregunta no sea contestada, abrí una conversación nueva

**Reply by Lautymendez** [reply]
Listo!

**Reply by Lautymendez** [reply]
Perfecto, la proxima vez abro una conversacion nueva. Use esta conversacion porque era del mismo tema pero para la proxima ya se

**Reply by martinvilu** [reply]
Es correcto, pero con cuidado si la pregunta _ya fue contestada_ puede que quede colgado

**Reply by jingusci** [reply]
Una pregunta @martinvilu, perdón que me meta en la consulta de alguien mas.
Veo que mostras un formulario para entregar, no pude encontrar donde está indicado que eso es la forma de entregar (seguramente me perdí de algo)

Eso es para todas las comisiones?

**Reply by martinvilu** [reply]
Hilo propio mejor, pero fue explicado en la clase del miércoles

**Reply by martinvilu** [reply]
Y el enlace está tambien en el campus
